.github/workflows/sdk-binary-size.yml

```diff
@@ -0,0 +1,67 @@
+name: SDK Binary Size
+# Only run this workflow on pull requests
+on: [pull_request]
+
+jobs:
+  sdk-binary-size-comparison-report:
+    name: Generate Comparison Report
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Android SDK
+        uses: ./.github/actions/setup-android
+
+      - name: Checkout base branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.base_ref }}
+
+      - name: Generate report for base branch
+        run: |
+          ./gradlew publishToMavenLocal
+          ./gradlew generateSdkSizeReport
+        env:
+          # Use local version to make sure the report is generated for the current changes
+          MODULE_VERSION: 'local'
```

Comment:
if we have `IS_DEVELOPMENT` true, wouldn't it automatically set the version to local?

Reply:
Looking at the code, this seems correct. I haven't tried it yet, will update and see if it works fine 👍🏻